openhtf/output/callbacks/mfg_inspector.py

```diff
@@ -238,7 +238,7 @@ def _extract_parameters(record, testrun, used_parameter_names):
         # Just a plain ol' value.
         if isinstance(value, numbers.Number):
           testrun_param.numeric_value = float(value)
-        else:
+        elif measurement.measured_value.is_value_set:
```

Comment:
Hm, I think I prefer to tweak this a little farther up, around line 231:
```py
if not measurement.measured_value:
  testrun_param.status = test_runs_pb2.ERROR
  continue
if not measurement.measured_value.is_value_set:
  testrun_param.status = test_runs_pb2.FAIL
  # Don't continue here, we'll generate an 'Unset' detail code below.
else:
  value = measurement.measured_value.value
  if measurement.dimensions is None:
    # ...
    # Indent all the way to line 270.
if testrun_param.status == test_runs_pb2.FAIL:
```
And if you're bored while you're here, you might pull some of these guts out into a helper function, this function is a bit too long to be easily digestible (my fault, sorry :( ).